tools/yaml-templates/build-app-host.yml

```diff
@@ -73,6 +73,13 @@ steps:
     workingDirectory: '$(AppHostingSdkProjectDirectory)'
     condition: eq(variables.CYPRESS_CACHE_RESTORED, 'false')
 
+  - task: CodeQL3000Init@0
```

Comment:
Web is running on Linux and I think it is fine to keep it as-is.

Reply:
@KangxuanYe is right here, this change is unnecessary.

Reply:
We need this change here, otherwise the codeQL will not run for web repo because the pipeline level codeQL skipping. It is also the same reason as I explained on "CodeQL is disabled at pipeline level" for the other two comments.

This test build should be able to explain better. https://office.visualstudio.com/ISS/_build/results?buildId=34120234&view=logs&j=dd9cb201-f7ed-50bd-a8c9-517b9a22558e&t=0adf847b-690e-5ae5-3253-69f3bf6c88d1 
As we can see, "CodeQl initialize" will always be skipped. The the only way to run scan on web repo is to add this task manually, which you can see the job "CodeQL3000Init@0" in the screen shot
![image](https://github.com/user-attachments/assets/54e511e1-aee5-4712-a54c-57fd3932e8e7)